userland/libtock/led.c

```diff
@@ -1,18 +1,20 @@
-#include "led.h"
-#include "tock.h"
+#include "internal/led.h"
+
+// For LEDs, these simply pass through to the syscall wrappers (the functions
+// in the internal folder).
```

Comment:
I was under the impression that was the structure you wanted to move to. I can't remember exactly where it was said, or if it was said explicitly, but it started in #419. Why do it for only timers? How does one know when to create an internal header file?